### PR TITLE
Remove title, license, contact and toc from the swagger-ui generated API docs

### PR DESCRIPTION
**Pull request by ayshsandu**

### CODE REVIEW

File: en/docs/concepts/oauth2/intro-oauth2.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ usable. When working with the tokens it guarantees API security by relying on Se
 data between the web server and browsers remain private. If a user can log in to a single OAuth 2.0 provider then he can
 log in to other web services without login to those web services differently.
 
-### When to choose OAuth2.0 is for which application ?
+### When to choose OAuth2.0 is for which application?
```
Comment (sherenewso2) [inline en/docs/concepts/oauth2/intro-oauth2.md]: Shouldn't this be "Which OAuth2.0 grant type to choose"?

File: en/docs/concepts/oauth2/intro-oauth2.md [hunk 2]
```diff
@@ -28,17 +28,18 @@ These different grant types target different application types.
 | Flow                 | Usage         | 
 | --------------------- | ------------- | 
 | Authorization code grant | Mostly recommended for web applications and single-page applications(SPA). This method is very safe in web applications, as tokens are passed directly to the web server and stored there without exposing the tokens to the resource owners. For Single page applications (JavaScript clients) it is recommended to use this grant type with PKCE.  |                            
-| Client credential grant  | Recommended for machine to machine communications (CLIs), if a machine needs to access a resource without human interaction, this grant type is recommended to use.  |                              
-| Implicit grant            | Used for single-page applications (SPA) to retrieve the access tokens directly without spending a number of round trips. Anyway, there is a security consideration, as the access token is exposed to the client-side.  | 
+| Client credential grant  | Recommended for a machine to machine communications (CLIs), if a machine needs to access a resource without human interaction, this grant type is recommended to use.  |
```
Comment (sherenewso2) [inline en/docs/concepts/oauth2/intro-oauth2.md]: The "a" should not be here. 

this grant type is recommended to use --> "this grant type is recommended." or "it is recommended to use this grant type"

File: en/docs/concepts/oauth2/intro-oauth2.md [hunk 3]
```diff
@@ -28,17 +28,18 @@ These different grant types target different application types.
...
-| Client credential grant  | Recommended for machine to machine communications (CLIs), if a machine needs to access a resource without human interaction, this grant type is recommended to use.  |                              
-| Implicit grant            | Used for single-page applications (SPA) to retrieve the access tokens directly without spending a number of round trips. Anyway, there is a security consideration, as the access token is exposed to the client-side.  | 
+| Client credential grant  | Recommended for a machine to machine communications (CLIs), if a machine needs to access a resource without human interaction, this grant type is recommended to use.  |                              
+| Implicit grant            | Used for single-page applications (SPA) to retrieve the access tokens directly without spending several round trips. Anyway, there is a security consideration, as the access token is exposed to the client-side.  |
```
Comment (sherenewso2) [inline en/docs/concepts/oauth2/intro-oauth2.md]: Anyway --> However?

File: en/docs/concepts/intro-authentication.md [hunk 4]
```diff
@@ -1,19 +1,22 @@
 # Introduction to Authentication
 
 ### User Identity
-The way in which a user can be uniquely identified is called ‘User Identity’. A user’s identity can differ on the 
-platform and the use case. Considering the digital world a single user can have different identities as user name, user 
-id, biometric information, etc.
+The way in which a user can be uniquely identified is called ‘User
+Identity’. A user’s identity can differ on the platform and the use
+case. In the digital world a single user can have different identities
+as user name, user id, email id, biometric information and etc.
```
Comment (sherenewso2) [inline en/docs/concepts/intro-authentication.md]: different identities as --> different identities such as

Comment (sherenewso2) [reply en/docs/concepts/intro-authentication.md]: id --> ID

Comment (sherenewso2) [reply en/docs/concepts/intro-authentication.md]: biometric information and etc. --> biometric information, etc.

Comment (sherenewso2) [inline en/docs/concepts/intro-authentication.md]: The way in which a user **is** uniquely identified is called **a** ‘User
Identity’. A user’s identity can differ **based** on the platform and the use
case. In the digital world{insert comma} a single user can have different **identifying attributes**
**such** as user name, user **ID**, email **ID**, biometric information**,** etc.

File: en/docs/concepts/intro-authentication.md [hunk 5]
```diff
@@ -1,19 +1,22 @@
...
-should be shared and stored with the system. When a user needs to access the system later, the system checks and verifies
-whether it is really the particular user who is trying to access the system by using that shared identity. It can be 
-introduced several factors of authentication based on the security level the server or the system needs to achieve.
+Simply **Authentication** is validating the user’s identity. In the user
+on-boarding process, the user’s identity should be shared and stored
+with the system. When a user needs to access the system later, the
+system checks and verifies whether it is really the particular user who
+is trying to access the system by using that shared identity. Several
+factors for authentication can be applied based on the security level
+the server or the system needs to achieve.
```
Comment (sherenewso2) [inline en/docs/concepts/intro-authentication.md]: Simply, **Authentication** is validating the user’s identity. **During** the user
on-boarding process, the user’s identity **is** shared and stored
**within** the system. When a user needs to access the system later, the
system checks and verifies against the previously stored identity information to confirm that it is authenticating the correct user. Several
factors for authentication can be applied based on the security level **that**
the server or the system needs to achieve.

File: en/docs/concepts/intro-protocols.md [hunk 6]
```diff
@@ -5,9 +5,12 @@ A protocol is a set of pre-defined rules used when two or multiple parties are c
...
 ### Password authentication protocol
-This is one of the oldest protocols used in the authentication. When using this protocol the authentication happens 
-using typical user-name and password approach. Even though this approach is simple to use and deploy it has some security
-considerations. If a malicious user is able to guess or obtain the password of a legitimate user, he can access the system.
+This is one of the oldest protocols used in the authentication. When
+using this protocol the authentication happens using typical **username
+and password** approach. Even though this approach is simple to use and
+deploy it has some security considerations. If a malicious can able to
+guess or obtain the password of a legitimate user, he can access the
+system.
```
Comment (sherenewso2) [inline en/docs/concepts/intro-protocols.md]: The password authentication protocol is one of the oldest protocols used **during** authentication. When
using this protocol, the authentication happens using the typical **username
and password** approach. Even though this approach is simple to use and
deploy, it has some security considerations. If a malicious **user** is able to
guess or obtain the password of a legitimate user, **he/she will be able to** access the
system.

File: en/docs/concepts/oauth2/intro-oauth2.md (hunk 1 above)
Comment (ayshsandu) [reply en/docs/concepts/oauth2/intro-oauth2.md]: fixed

File: en/docs/concepts/oauth2/intro-oauth2.md (hunk 2 above)
Comment (ayshsandu) [reply en/docs/concepts/oauth2/intro-oauth2.md]: different identities such as

File: en/docs/concepts/oauth2/intro-oauth2.md (hunk 3 above)
Comment (ayshsandu) [reply en/docs/concepts/oauth2/intro-oauth2.md]: fixed

File: en/docs/concepts/intro-authentication.md (hunk 4 above)
Comment (ayshsandu) [reply en/docs/concepts/intro-authentication.md]: @DMHP, please node

File: en/docs/concepts/intro-authentication.md (hunk 5 above)
Comment (ayshsandu) [reply en/docs/concepts/intro-authentication.md]: @DMHP, please note

File: en/docs/concepts/intro-protocols.md (hunk 6 above)
Comment (ayshsandu) [reply en/docs/concepts/intro-protocols.md]: @DMHP please note

**Pull request merged**